Vagrantfile

```diff
@@ -0,0 +1,44 @@
+# -*- mode: ruby -*-
```

Comment:
I am not familiar with Vagrant. Can you give me some background? I would love to learn a little.